src/js/actions/menu.js

```diff
@@ -570,6 +570,181 @@ define(function (require, exports) {
         writes: [locks.JS_PREF],
         transfers: [preferencesActions.setPreference]
     };
+    
+    /**
+     * Debug-only method to toggle Descriptor Event logging
```

Comment:
I suggest we split out all the actions currently in `menu` that are only relevant to debugging and put them in a separate action module. For bonus points, it would be great if we could not load those actions if `__PG_DEBUG__` is not set! But, even if we don't go that far in this PR, it would still be nice to streamline the menu actions.